packages/app/src/templates/services/template.service.test.ts

```diff
@@ -111,4 +112,35 @@ describe("`TemplateService`", () => {
       expect(response.status).toEqual(RequestResponseStatus.Error)
     })
   })
+  describe("`updateTemplateOrder` method", () => {
+    test("map data and returns success when `deviceService.request` returns success", async () => {
+      deviceService.request = jest.fn().mockReturnValue(successResponse)
+      const response = await subject.updateTemplateOrder(template)
+      expect(deviceService.request).toHaveBeenLastCalledWith({
+        endpoint: 8,
+        method: 3,
+        body: {
+          templateID: 1,
+          category: MessagesCategory.template,
+          order: 1,
+        },
+      })
```

Comment:
Maybe would be better to use consts for e.g. `endpoint`, `method` etc.?

Reply:
Added in [21cf63a](https://github.com/mudita/mudita-center/pull/798/commits/21cf63a04a2ea86dd5b4695e89115b58a9e7991b)